field/bitmap.go

```diff
@@ -170,15 +181,24 @@ func (f *Bitmap) bitmapsCount() int {
 	return count
 }
 
+func (f *Bitmap) getMapSize() int {
+	if f.mapSize == 0 {
+		f.mapSize = defaultBitmaps
```

Comment:
I think also a `f.Reset()` after this as well just in case - i realize that in practice that it will probably be used in conjuction with `setMapSize` which will call `f.Reset` but to me it feels a bit 'risky' to have a bitmap say it has size 1 but with no actually initialized values

Reply:
Yes, 
the code has problem when using Bitmap directly